lightning/src/ln/channel.rs

```diff
@@ -2098,8 +2139,22 @@ impl<Signer: Sign> Channel<Signer> {
 			}
 		}
 
+		if msg.amount_msat / 1000 < (self.get_dust_buffer_feerate() as u64 * HTLC_TIMEOUT_TX_WEIGHT / 1000) + self.counterparty_dust_limit_satoshis {
+			if inbound_stats.on_counterparty_tx_dust_balance + outbound_stats.on_counterparty_tx_dust_balance + msg.amount_msat > self.get_max_balance_dust_htlc_msat() {
+				log_info!(logger, "Cannot accept value that would put holder dusted balance {} on counterparty commitment over limit {}", inbound_stats.on_counterparty_tx_dust_balance + outbound_stats.on_counterparty_tx_dust_balance + msg.amount_msat, self.get_max_balance_dust_htlc_msat());
```

Comment:
"dusted" is really not a sensible word for a user, I think. Maybe "Cannot accept value that would put our exposure to dust HTLCs over the limit", plus all the other logs in this PR.

nit: can you split this line by moving some of the parameters to the next line with an extra indent?